boat/commands/guideline.js

```diff
@@ -47,10 +59,19 @@ module.exports = async function (msg, args) {
     }
 
     const guideline = match[0].slice(2).replace(/\n\n/g, '<br><br>').split('\n')
-    guideline[0] = `**Guideline #${guideline[0]}**\n`
-    const parts = guideline.map(g => g.replace(/<br>/g, '\n'))
-    msg.channel.createMessage(`${parts.join('').trim()}\n\n${INFO_STR}`)
+    const embed = {
+      title: guideline.shift(),
+      description: guideline.map(g => g.replace(/<br>/g, '\n')).join('').trim(),
+      fields: [
+        {
+          name: 'Read all the guidelines',
+          value: 'https://powercord.dev/guidelines'
+        }
+      ]
+    }
+    msg.channel.createMessage({ embed })
   } catch (e) {
+    console.log(e)
```

Comment:
It'd be preferable to log it to stderr and give it a bit of context
```suggestion
    console.error('error occurred while fetching guidelines', e)
```